### DISCUSSION TITLE
Create one method for BatchUpdate.

### DISCUSSION THREAD

**Discussion by parthchat99** [sensedeep/dynamodb-onetable#478] [Ideas]
I am trying to update more than one field and since there is no option of batchUpdate, I have to update every field using for loop. Besides performance, it is costing me more on AWS

**Comment by mobsense** [comment]
Can you share your code snippet?

You should be able to update multiple fields in one update call -- for a single item. 

If you are updating multiple items, you would have the same overhead with OneTable or with DynamoDB directly.
